src/main/java/com/puppycrawl/tools/checkstyle/checks/design/DesignForExtensionCheck.java

```diff
@@ -176,44 +176,76 @@
  * &lt;module name=&quot;DesignForExtension&quot;/&gt;
  * </pre>
  * <p>
- * To configure the check to allow methods which have @Override and @Test annotations
+ * Example:
+ * </p>
+ * <pre>
+ * public class A {
+ *
+ *     &#64;BeforeClass
+ *     public int beforeAllTests() { return 1; } // OK, @BeforeClass is ignored by default
+ *
```

Comment:
Please make methods to be void to keep body as `{}`

Reply:
Please make indentation to be +2 space to make code more short, to fit small screens better
And user focus on what matters.

Reply:
https://github.com/checkstyle/checkstyle/pull/12353#discussion_r1010020894

If I make methods void, Checkstyle stops giving violations on those particular functions when I run it. I wonder why?

Reply:
https://github.com/checkstyle/checkstyle/pull/12353#discussion_r1010020894

If I make methods void, Checkstyle stops giving violations on those particular methods because making methods empty would cause it not to be evaluated by DesignForExtension check
![image](https://user-images.githubusercontent.com/103727912/199775767-13068604-c41a-4d3a-9e67-fba150ea0f4f.png)

Reply:
Ok, keep int.